src/core/components/radio/Radio.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { ReactNode, InputHTMLAttributes } from 'react';
+import {
+	label,
+	labelWithSupportingText,
+	radio,
+	labelText,
+	labelTextWithSupportingText,
+	supportingText,
+} from './styles';
+import { Props } from '@guardian/src-helpers';
+
+const LabelText = ({
+	hasSupportingText,
+	children,
+}: {
+	hasSupportingText?: boolean;
+	children: ReactNode;
+}) => {
+	return (
+		<div
+			css={(theme) => [
+				hasSupportingText ? labelTextWithSupportingText : '',
+				labelText(theme.radio && theme),
+			]}
+			className="src-radio-label-text"
+		>
+			{children}
+		</div>
+	);
+};
+
+const SupportingText = ({ children }: { children: ReactNode }) => {
+	return (
+		<div css={(theme) => supportingText(theme.radio && theme)}>
+			{children}
+		</div>
+	);
+};
+
+export interface RadioProps
+	extends InputHTMLAttributes<HTMLInputElement>,
+		Props {
+	/**
+	 * TODO: Add a description or remove as value is inherited from `InputHTMLAttributes<HTMLInputElement>` (as `string | number | readonly string[] | undefined`)
+	 */
+	value: string;
+	/**
+	 * Whether radio button is checked. This is necessary when using the [controlled approach](https://reactjs.org/docs/forms.html#controlled-components) to form state management.
+	 *
+	 * _Note: if you pass the `checked` prop, you MUST also pass an `onChange` handler, or the field will be rendered as read-only._
+	 */
+	checked?: boolean;
+	/**
+	 * TODO: Add a description
+	 */
+	defaultChecked?: boolean;
```

Comment:
Is "Whether radio button is checked initially" a suitable description? Is it best practice to only use this with the uncontrolled approach?